### PR TITLE
Migrating oracle unit tests job from jenkins to drone

### PR DESCRIPTION
**Pull request by SergioBertolinSG**
Work in progress

Pending:

- [x] Fix oracle port in wait-for-it part.
- [x] Add oci Extension to PHP dockers.
- [x] Remove the start/stop of oracle container in unit test execution.

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
@ownclouders rebase

**Comment by DeepDiver1975** [comment]
@ownclouders rebase

**Comment by ownclouders** [comment]
Hey! This pull request is being rebased automatically. Please DO NOT push while rebase is in progress or your changes would be lost!

**Comment by ownclouders** [comment]
Hey! This pull request is being rebased automatically. Please DO NOT push while rebase is in progress or your changes would be lost!

**Comment by ownclouders** [comment]
Automated rebase was successful!

File: tests/drone/test-phpunit.sh [hunk 1]
```diff
@@ -32,6 +32,10 @@ else
       wait-for-it postgres:5432
       DB=pgsql
       ;;
+    oracle)
+      wait-for-it oracle:1521
```
Comment (DeepDiver1975) [inline tests/drone/test-phpunit.sh]: this is also not enough - the port will be available pretty early - but during the init phase the database will throw an error back .... see build/autotest.sh here as well

**Review by DeepDiver1975 — CHANGES_REQUESTED** [review]

File: tests/drone/test-phpunit.sh (hunk 1 above)
Comment (SergioBertolinSG) [reply tests/drone/test-phpunit.sh]: OK. I'm trying to use sqlplus but the loop doesn't end:

```bash
      DATABASEUSER=autotest
      DATABASENAME='XE'
      for i in {1..48}
          do
              if "${SQLPLUS}" "${DATABASEUSER}/owncloud@(DESCRIPTION=(ADDRESS=(PROTOCOL=TCP)(Host=$DATABASEHOST)(Port=1521))(CONNECT_DATA=(SID=XE)))" < /dev/null | grep 'Connected to'; then
                  break;
              fi
              sleep 5
          done
```

is that line with the connection correct?

Comment (SergioBertolinSG) [reply tests/drone/test-phpunit.sh]: Is that really needed? We don't have it in mysql and oracle tests here have passed just checking the port.

Comment (DeepDiver1975) [reply tests/drone/test-phpunit.sh]: mysql !== oracle

**Review by DeepDiver1975 — APPROVED** [review]

**Comment by PVince81** [comment]
backport to stable10?

**Comment by phil-davis** [comment]
Backport stable10 #30066

**Pull request merged**